pkg/gardenlet/controller/shoot/shoot/reconciler_delete.go

```diff
@@ -246,7 +245,7 @@ func (r *Reconciler) runDeleteShootFlow(ctx context.Context, o *operation.Operat
 				waitUntilEtcdReady,
 				waitUntilKubeAPIServerServiceIsReady,
 				waitUntilControlPlaneReady,
-			).InsertIf(!staticNodesCIDR),
+			),
```

Comment:
I don't understand why this check was there and why it is removed now...

Reply:
The signature of that method is `InsertIf(condition bool, iders ...TaskIDer)`. Apparently someone removed the task list without cleaning up the now obsolete call to `InsertIf`)

Reply:
ah, I thought it inserts the previous conditions conditionally... Then it makes sense to clean this up